src/main/java/nextstep/subway/domain/SubwayMap.java

```diff
@@ -41,7 +41,7 @@ public Path findPath(Station source, Station target, PathType pathType) {
                 .forEach(it -> {
                     SectionEdge sectionEdge = SectionEdge.of(it);
                     graph.addEdge(it.getUpStation(), it.getDownStation(), sectionEdge);
-                    graph.setEdgeWeight(sectionEdge, it.getPathTypeValue(pathType));
+                    graph.setEdgeWeight(sectionEdge, getPathTypeValue(it, pathType));
```

Comment:
피드백 반영 👍 
pathType 에게 메시지를 보내도 좋을 것 같은데 어떻게 생각하시나요?
```suggestion
                    graph.setEdgeWeight(sectionEdge, pathType.value(it));
```

Reply:
PathType의 값에 따라 반환되는 로직이기 때문에 관련된 PathType에 위치하는게 더  좋을 것 같습니다! 수정하겠습니다!